src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/PackageLibs.targets

```diff
@@ -140,6 +140,12 @@
       <FilesToPackage Condition="'%(FilesToPackage.SubFolder)' != ''">
         <TargetPath>%(TargetPath)%(FilesToPackage.SubFolder)</TargetPath>
       </FilesToPackage>
+
+      <!-- Include source files for symbol packages. -->
+      <FilesToPackage Include="@(Compile->'%(FullPath)')" Condition="$([System.String]::new('%(FullPath)').StartsWith('$(ProjectDir)'))">
+        <TargetPath>src\$([System.String]::new('%(FullPath)').Replace('$(ProjectDir)', ''))</TargetPath>
```

Comment:
Just wish this didn't use string comparison, that feels fragile and the mode of failure is to just drop what isn't understood.  Seems to me we should fix the unix behavior of URI...
Barring that I'd say lets have the inverse of this condition drop the src files in the root src\ directory.  Also, use substring rather than replace since you already know the thing you are removing is at the start of the string.  Can you double check that ProjectDir will always be normalized, otherwise create a new property that you've normalized with GetFullPath.